pwndbg/commands/got.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pwndbg.chain
+import pwndbg.checksec
+import pwndbg.commands
+import pwndbg.enhance
+import pwndbg.file
+import pwndbg.which
+
+from pwndbg.color import green
+from pwndbg.color import light_yellow
+
+
+@pwndbg.commands.Command
+@pwndbg.commands.OnlyWhenRunning
+@pwndbg.commands.OnlyWithFile
+def got():
+    '''
+    Show the state of the Global Offset Table
+    '''
+
+    local_path = pwndbg.file.get_file(pwndbg.proc.exe)
+    cs_out = pwndbg.checksec.checksec(local_path)
+
+    file_out = pwndbg.file.file(local_path)
+    if "statically" in file_out:
+        return "Binary is statically linked."
+
+    jmpslots = pwndbg.elf.getjmpslots(local_path)
+    if not len(jmpslots):
+        return "NO JUMP_SLOT entries available in the GOT"
+
+    if cs_out['PIE']:
+        bin_text_base = pwndbg.memory.page_align(pwndbg.elf.entry())
+
+    relro_status = "No RELRO"
+    if cs_out['RELRO'] == 2:
+        relro_status = "Full RELRO"
```

Comment:
It looks like we're converting between `"Full RELRO"` to `2` back to `"Full RELRO"`.  Since the output of `checksec` may change, perhaps it's just best to avoid this conversion and capture the output?